tests/unit/test-cmd/test.build.js

```diff
@@ -90,7 +92,12 @@ describe('build', () => {
     );
   });
 
-  it('checks locale file for malformed json', () => {
+  // FIXME: Failed on Node 7.
+  // https://github.com/sindresorhus/parse-json/issues/6
+  it('checks locale file for malformed json', function() {
+    if (semver.gte(process.versions.node, '7.0.0')) {
+      return this.skip();
```

Comment:
We try to stick to [LTS](https://github.com/nodejs/LTS) pretty closely and Node 7 isn't there yet. I'd rather let this fail for now since we don't officially support Node 7.